File: frontend/src/app/master/models/metricsMaster.ts
export interface MetricsMaster {
  id: number | null;                  
  metricsName: string;                
  description: string | null;         
  startDate: string | null;             
  endDate: string | null;               
  remarks: string | null;             
  active?: boolean;                   
  status: number;                     
  createdBy?: string |null;                 
  modifiedBy?: string|null;                
}

File: backend/Models/MetricsMaster.cs
using System.ComponentModel.DataAnnotations;

namespace backend.Models
{
    public class MetricsMaster : BaseModel
    {
        [Key]
        public int Id { get; set; }
        public string? MetricsName { get; set; }
        public string? Description { get; set; }
        //public int? ProjectTypeId { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public string? Remarks { get; set; }
    }
}
